charts/authentik/values.yaml

```diff
@@ -118,175 +195,740 @@ authentik:
+
+    # -- uses `server.service.servicePortHttps` instead of `server.service.servicePortHttp`
+    https: false
+
+
+## authentik worker
+worker:
+  # -- authentik worker name
+  name: worker
+
+  # -- The number of worker pods to run
+  replicas: 1
+
+  ## authentik worker Horizontal Pod Autoscaler
+  autoscaling:
+    # -- Enable Horizontal Pod Autoscaler ([HPA]) for the authentik worker
+    enabled: false
+    # -- Minimum number of replicas for the authentik worker [HPA]
+    minReplicas: 1
+    # -- Maximum number of replicas for the authentik worker [HPA]
+    maxReplicas: 5
+    # -- Average CPU utilization percentage for the authentik worker [HPA]
+    targetCPUUtilizationPercentage: 50
+    # -- Average memory utilization percentage for the authentik worker [HPA]
+    targetMemoryUtilizationPercentage: ~
+    # -- Configures the scaling behavior of the target in both Up and Down directions.
+    behavior: {}
+      # scaleDown:
+      #   stabilizationWindowSeconds: 300
+      #   policies:
+      #     - type: Pods
+      #       value: 1
+      #       periodSeconds: 180
+      # scaleUp:
+      #   stabilizationWindowSeconds: 300
+      #   policies:
+      #     - type: Pods
+      #       value: 2
+      #       periodSeconds: 60
+    # -- Configures custom HPA metrics for the authentik worker
+    # Ref: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
+    metrics: []
+
+  ## authentik worker Pod Disruption Budget
+  ## Ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/
+  pdb:
+    # -- Deploy a [PodDistrubtionBudget] for the authentik worker
+    enabled: false
+    # -- Labels to be added to the authentik worker pdb
+    labels: {}
+    # -- Annotations to be added to the authentik worker pdb
     annotations: {}
     # -- Number of pods that are available after eviction as number or percentage (eg.: 50%)
     # @default -- `""` (defaults to 0 if not specified)
     minAvailable: ""
     # -- Number of pods that are unavailable after eviction as number or percentage (eg.: 50%)
-    ## Has higher precedence over `pdb.worker.minAvailable`
+    ## Has higher precedence over `worker.pdb.minAvailable`
     maxUnavailable: ""
 
-livenessProbe:
-  # -- enables or disables the livenessProbe
-  enabled: true
-  httpGet:
-    # -- liveness probe url path
-    path: /-/health/live/
-    port: http
-  initialDelaySeconds: 5
-  periodSeconds: 10
-
-startupProbe:
-  # -- enables or disables the livenessProbe
-  enabled: true
-  httpGet:
-    # -- liveness probe url path
-    path: /-/health/live/
-    port: http
-  failureThreshold: 60
-  periodSeconds: 5
-
-readinessProbe:
-  enabled: true
-  httpGet:
-    path: /-/health/ready/
-    port: http
-  periodSeconds: 10
+  ## authentik worker image
+  image:
+    # -- Repository to use to the authentik worker
+    # @default -- `""` (defaults to global.image.repository)
+    repository: "" # defaults to global.image.repository
```

Comment:
I would agree, but for SaaS, we might want to deploy the worker first, see if it's running, and then update the server (with something like Argo Rollouts or other, idk). The goal here was to provide us with as much flexibility as possible, while also allow other users to write as little as they need. What we could do though, is remove the comments so that it doesn't end up in the README.

Reply:
That's also the reason why there are a lot of those under `global`

Reply:
We could also add another value that needs to be set to `true` or something to make it more explicit that this isn't something that most people should do

Reply:
something like `server.image.overwrite`?